edgeboxctl.service

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=Edgebox Control Module Service
+ConditionPathExists=/home/system/
+After=network.target
+ 
+[Service]
+Type=simple
+User=root
+Group=root
+LimitNOFILE=1024
+
+Restart=on-failure
+RestartSec=10
+startLimitIntervalSec=60
+
+WorkingDirectory=/home/system/components/edgeboxctl

```

Comment:
As far as I'm aware, edgeboxctl can run without needing to be in a specific work directory. Have to double check.